docs/components/tree-list.md

```diff
@@ -0,0 +1,330 @@
+# Tree List
+
+**KTreeList** - A drag-n-drop reorderable list component.
+
+::: tip
+It can be tricky to determine if an item will be places below an item as a sibling or as a child when dropping. Items to be dropped as children should be dropped near the bottom border line of the parent. If this will be the first child of an item, you will notice the margin between the two items disappearing if you are hovering in the right place.
+:::
+
+<KTreeList :items="defaultItems" />
+
+```html
+<KTreeList :items="items" />
+```
+
+## Props
+
+### items
+
+An array of items that make up the tree.
+
+<KTreeList :items="defaultItems2" />
+
+```html
+<template>
+  <KTreeList :items="items" />
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+  data() {
+    return {
+      items: [{
+        name: "Cats",
+        id: 'cats'
+      },
+      {
+        name: "Dogs",
+        id: 'dogs',
+        children: [{
+          name: "Puppies",
+          id: 'puppies'
+        }]
+      },
+      {
+        name: "Bunnies",
+        id: 'bunnies'
+      }]
+    }
+  }
+})
+</script>
+```
+
+### disableDrag
+
+Boolean (defaults to `false`) to turn of drag-n-drop reordering of the list.
+
+<KTreeList disable-drag :items="disableItems" />
+
+```html
+<KTreeList disable-drag :items="items" />
+```
+
+### maxLevel
+
+Use this prop to customize the maximum supported depth of the tree. We default to a max level of `2` or support for parents, children, and grandchildren.
+
+<KTreeList :items="maxLevelItems" :max-level="1" />
+
+```html
+<KTreeList :items="items" :max-level="1" />
+```
+
+### v-model
```

Comment:
If `v-model` usage is preferred, I think it should be listed above the `items` prop